doc/introduction/templates.rst

```diff
@@ -228,6 +228,7 @@ The following initialization functions are available:
 .. autosummary::
     :nosignatures:
 
+<<<<<<< HEAD
```

Comment:
more merge issues here

Reply:
Very sorry, removed and made sure that there are no merge-like things in the entire path...